lib/measures/tbd/measure.rb

```diff
@@ -308,7 +308,7 @@ def run(user_model, runner, user_arguments)
       io_path = runner.workflow.findFile('tbd.json')
       if io_path.empty?
         TBD.log(TBD::FATAL, "Can't find 'tbd.json' - simulation halted")
-        return exitTBD(runner)
+        return exitTBD(user_model, runner)
```

Comment:
@macumber : This needed fixing once the measure tests were enabled - a minimum of 2 arguments are needed here.